lib/brainstem/api_docs/formatters/open_api_specification/version_2/endpoint/param_definitions_formatter.rb

```diff
@@ -0,0 +1,272 @@
+                    'type'        => 'integer',
+                    'description' => "The ID of the #{model_name.humanize}."
+                  }
+                end
+              end
+
+              def path_params
+                endpoint.path
+                  .gsub('(.:format)', '')
+                  .scan(/(:(?<param>\w+))/)
+                  .flatten
+              end
+
+              def nested_properties(param_config)
+                param_config.except(:_config)
+              end
+
+              def format_pagination_params!
+                output << format_query_param(:page, type: 'integer', default: 1)
+                output << format_query_param(:per_page, type: 'integer', default: 20, maximum: 200)
+              end
+
+              def format_search_param!
+                return if presenter.nil? || !presenter.searchable?
+
+                output << format_query_param(:search, type: 'string')
+              end
+
+              def format_only_param!
+                output << format_query_param(:only,
+                  type: 'string',
+                  info: 'Allows you to request one or more resources directly by IDs in a comma separated list'
+                )
+              end
+
+              def format_sort_order_params!
+                return if presenter.nil? || (valid_sort_orders = presenter.valid_sort_orders).empty?
+
+                sort_orders = valid_sort_orders.map { |sort_name, _|
+                  ["#{sort_name}:asc", "#{sort_name}:desc"]
+                }.flatten.sort
+
+                output << format_query_param('order',
+                  info:      'Supply `order` with the name of a valid sort field for the endpoint and a direction',
+                  type:      'array',
+                  item_type: 'string',
+                  items:     sort_orders,
+                  default:   presenter.default_sort_order
+                )
+              end
+
+              def format_optional_params!
+                return if presenter.nil? || (optional_field_names = presenter.optional_field_names).empty?
+
+                output << format_query_param('optional_fields',
+                  info:      'Allows you to request one or more optional fields as an array',
+                  type:      'array',
+                  item_type: 'string',
+                  items:     optional_field_names
+                )
+              end
+
+              def format_include_params!
+                return if presenter.nil? || presenter.valid_associations.empty?
+
+                output << format_query_param('include',
+                  type: 'string',
+                  info: include_params_description
+                )
+              end
+
+              def include_params_description
+                result = "Any of the below associations can be included in your request by providing the `include` "\
+                         "param, e.g. `include=association1,association2.`\n"
```

Comment:
The period (`.`) should be outside the code formatting.

Reply:
Also, shouldn't this use `md_inline_code`?

Reply:
Using inline code seems a bit unnecessary here since it's all text.